### PR TITLE
demosite: smaller records

### PR DESCRIPTION
**Pull request by kaplun**
Signed-off-by: Samuele Kaplun <samuele.kaplun@cern.ch>

### CODE REVIEW

**Comment by kaplun** [comment]
Let's see which test fail.

**Comment by jacquerie** [comment]
How many records are there? If it's small enough we might even consider not gzipping it...

**Comment by kaplun** [comment]
62 records. 4MB I think it's still nice to keep it gzipped with that size :)

File: tests/integration/hal/fixtures/test_tei_record.xml [hunk 1]
```diff
@@ -44,7 +44,16 @@
...
+                <monogr>
+                      <title level="j">Adv.Ser.Direct.High Energy Phys.</title>
+                    <imprint>
+                        <biblScope unit="volume">26</biblScope>
+                        <biblScope unit="pp">231-253</biblScope>
+                        <date type="datePub">2016</date>
+                    </imprint>
+                </monogr>
 
+                <idno type="doi">10.1142/9789814733519_0013</idno>
```
Comment (jacquerie) [inline tests/integration/hal/fixtures/test_tei_record.xml]: Uh? I don't understand this change.

File: tests/integration/test_bibtex_exporting.py [hunk 2]
```diff
@@ -64,10 +64,14 @@ def test_format_proceeding(app):
     proceeding = get_db_record('lit', 701585)
 
     expected = u'''@proceedings{Alekhin:2005dx,
-      author         = "Alekhin, S. and others",
+      author         = "De Roeck, A. and Jung, H.",
```
Comment (jacquerie) [inline tests/integration/test_bibtex_exporting.py]: This also looks weird.

Oh, I understand, the record change on INSPIRE in the meantime: https://inspirehep.net/record/701585.

File: tests/integration/test_datatable_citations.py [hunk 3]
```diff
@@ -28,14 +28,5 @@ def test_citations(app):
         response = client.get('/ajax/citations?recid=712925&endpoint=literature')
         assert response.status_code == 200
 
-        response = client.get('/ajax/citations?recid=1319638&endpoint=literature')
-        assert response.status_code == 200
-
         response = client.get('/ajax/citations?recid=452060&endpoint=literature')
         assert response.status_code == 200
```
Comment (jacquerie) [inline tests/integration/test_datatable_citations.py]: IMO you can also remove this guy, we only need to test one record here (and write a better test).

File: tests/integration/test_datatable_references.py [hunk 4]
```diff
@@ -26,14 +26,5 @@ def test_references(app):
         response = client.get('/ajax/references?recid=712925&endpoint=literature')
         assert response.status_code == 200
 
-        response = client.get('/ajax/references?recid=1319638&endpoint=literature')
-        assert response.status_code == 200
-
         response = client.get('/ajax/references?recid=452060&endpoint=literature')
         assert response.status_code == 200
```
Comment (jacquerie) [inline tests/integration/test_datatable_references.py]: Same as above.

File: tests/integration/test_detailed_records.py [hunk 5]
```diff
@@ -30,7 +30,7 @@
 def test_all_records_were_loaded(app):
     records = [record.json for record in RecordMetadata.query.all()]
 
-    expected = 576
+    expected = 42
```
Comment (jacquerie) [inline tests/integration/test_detailed_records.py]: \o/

File: tests/integration/hal/fixtures/test_tei_record.xml (hunk 1 above)
Comment (kaplun) [inline tests/integration/hal/fixtures/test_tei_record.xml]: Yes. That TEI record was completely enhanced after its first inclusion in the demosite.

File: tests/integration/test_bibtex_exporting.py (hunk 2 above)
Comment (kaplun) [inline tests/integration/test_bibtex_exporting.py]: Yup!

**Pull request merged**